### PR TITLE
Add note for php.frl user group talk

### PR DESCRIPTION
**Pull request by dbu**

### CODE REVIEW

File: src/Cmf/MainBundle/Resources/data/block.yml [hunk 1]
```diff
@@ -57,3 +57,10 @@ static:
             publishEndDate: "2015-06-25 9:30"
             link: http://www.phpconference.nl/schedule
             body: Symfony CMF Tutorial (3 hour workshop)
+    phpfrl-2015-leeuwarden:
+        class: Cmf\MainBundle\Document\TalkBlock
+        properties:
+            title: PHP.FRL Leeuwarden
+            publishEndDate: "2015-06-23 19:00"
+            link: http://www.meetup.com/PHP-FRL/events/222784422/
+            body: Symfony CMF Talk at the PHP user group friesland
```
Comment (dantleech) [inline src/Cmf/MainBundle/Resources/data/block.yml]: Should it be `Freisland` ?

Comment (dbu) [inline src/Cmf/MainBundle/Resources/data/block.yml]: friesland is the dutch name for it. according to wikipedia, its frisia
in english... http://en.wikipedia.org/wiki/Frisia#Names i guess dutch is ok.

Comment (dantleech) [inline src/Cmf/MainBundle/Resources/data/block.yml]: Sorry, my spelling mistake. I meant should it be capitalized?

Comment (dbu) [inline src/Cmf/MainBundle/Resources/data/block.yml]: indeed, that capitalization was off. Talk would not need to be
uppercase... fixed and pushed to master directly.

**Pull request merged**